content/en/docs/concepts/configuration/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+ 
+Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. At a very high level, Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. It models the scheduling problem as a constraint-based optimization over a flow network graph – by reducing scheduling to a min-cost max-flow optimization problem.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Introduction	
+
+Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. At a very high level, Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. It models the scheduling problem as a constraint-based optimization over a flow network graph – by reducing scheduling to a min-cost max-flow optimization problem.
+
+Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling environment that constantly keeps refining the workloads placements dynamically.
+
```

Comment:
This sentence could use some clarification. I find the last clause confusing. What is a globally optimal scheduling environment? And the environment constantly refines the placement of the workloads?

Reply:
The content has been modified and simplified.